tests/phpunit/zoteroTest.php

```diff
@@ -56,19 +56,55 @@ protected function expand_via_zotero($text) {
     $expanded->tidy();
     return $expanded;
   }
-  
-  public function testZoteroExpansion() {
+
+  public function testZoteroExpansionRGJunk() {
+    $text = '{{Cite journal|url =  https://www.researchgate.net/publication/2344536}}';
```

Comment:
The purpose of this test is not immediately apparent

Reply:
This is just a research gate page that gives rubbish title